options/m_config_core.c

```diff
@@ -31,13 +31,13 @@
 #include "common/msg.h"
 #include "common/msg_control.h"
 #include "misc/dispatch.h"
-#include "osdep/atomic.h"
+#include <stdatomic.h>
```

Comment:
this header should be moved to system headers section above, as per mpv's coding style.

(same for all other files)